### PR TITLE
Allow deletion of study annotators

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

**Comment by brianhelba** [comment]
Just need to fix the alignment of the buttons now.

**Comment by msmolens** [comment]
@brianhelba LGTM. Please take a look at the button changes in the most recent commit.

File: web_external/Studies/studyPage.jade [hunk 1]
```diff
@@ -18,18 +18,24 @@
...
+            title='Add annotator')
             i.icon-plus
     tr
       td.isic-study-users-content
         if study.get('users').length
-          ul
+          table(class=canAdminStudy ? 'table-hover' : undefined)
             each user in study.users().models
-              li= user.name()
+              tr
```
Comment (brianhelba) [inline web_external/Studies/studyPage.jade]: Can we put the `data-user-id=user.id` attribute up on the `tr`, so it's usable by the whole row in the future?

File: web_external/Studies/StudyView.js [hunk 2]
```diff
@@ -15,6 +15,14 @@ isic.views.StudyView = isic.View.extend({
             this.studyAddUserWidget.render();
         },
 
+        'click .isic-study-remove-user-button': function (event) {
+            var target = $(event.currentTarget);
+            var userId = target.parent().data('userId');
```
Comment (brianhelba) [inline web_external/Studies/StudyView.js]: If you move the `data-user-id`, this line will have to be:
```
var userId = target.closest('data-user-id').data('userId');
```

**Comment by brianhelba** [comment]
@msmolens Great change, this looks good, and the table hover helps even more.

I just think we should make the one tweak (described inline) to where the data is stored.

**Pull request merged**